example/samples/install_photon_payload_minimal.json

```diff
@@ -0,0 +1,8 @@
+{
+    "options": {
+        "defaults": {
+            "version": "1.0",
```

Comment:
Version 1.0 is a default in the task definition: https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/install-photon-os.js#L19, so it can be ignored in the minimal example.